src/code/init.js

```diff
@@ -140,6 +168,9 @@ window.plugin.wasabee.init = async () => {
   // run crosslinks
   window.map.fire("wasabeeCrosslinks", { reason: "startup" }, false);
 
+  // draw background ops
```

Comment:
Do background ops factor into crosslinks? My preference is for not having them trigger that extra slow-down.

Reply:
atm, background ops are not considered for crosslinks